README.md

```diff
@@ -102,11 +102,17 @@ Below should serve as an "upgrade guide" jumping off point as you are migrating
 - `?.?.?` - *WIP*
   - Updating linting, fix various spelling and linting errors
   - `lodash` bumped to `4.17.11`
+  - Ramda upgraded from 0.25.0 to 0.26.1
+    - Removed internal implementation of thunkify, now comes from Ramda
+    - Removed `evolveArray`, Ramda evolve now supports arrays with `evolve`
   - `stringify-object` bumped to `3.3.0`
   - Added methods:
     - `allSettledP`
     - `forEachSerialP`
     - `mapParallelLimitP`
+  - Removed methods:
```

Comment:
❌ add `thunkify` to the removed methods list here. The deeper explanation can live in the ramda upgrade bullet (or just removed or consolidated?). Blocking because I think the changelog is critical to keep up-to-date for future us.

Reply:
should I also add it to the added method list then, too? because it is still available, it is just not the "same"

Reply:
oh, my bad, yeah. We should just add at "changed methods" bullet.

Reply:
Yeah, sorry, I don't think this is a big deal because a lot changed with Ramda and we've already pointed "future us" to go look at those change logs.